django_stubs_ext/django_stubs_ext/patch.py

```diff
@@ -38,6 +39,7 @@ def __repr__(self) -> str:
     MPGeneric(ModelAdmin),
     MPGeneric(FormMixin),
     MPGeneric(BaseModelAdmin),
+    MPGeneric(Field),
```

Comment:
Maybe we can also provide an `extra=` argument to `.monkeypatch(extra=)` call?

So user will have a chance to also patch custom classes in case this is required.

Reply:
I wouldn't object, but I'm not sure I understand the use case.  I would say it should be a goal of django-stubs to patch everything covered by its typing implementation, and I don't see a use case for patching objects outside of the scope of django-stubs.

Do you have a specific use case in mind?

Reply:
No 🙂